addons/website_sale/static/src/js/tours/tour_utils.js

```diff
@@ -21,28 +21,28 @@ function assertCartAmounts({taxes = false, untaxed = false, total = false, deliv
     if (taxes) {
         steps.push({
             content: 'Check if the tax is correct',
-            trigger: `tr#order_total_taxes .oe_currency_value:containsExact(${taxes})`,
+            trigger: `tr#order_total_taxes .oe_currency_value:contains(/^${taxes}$/)`,
```

Comment:
How does the string work with hoot? Has the $ anything to do with the fact that it's a currency?

Reply:
Hello. Hoot :contains() pseudo selector accepts RegExp.